requestlog/requestlog_test.go

```diff
@@ -75,20 +76,41 @@ func TestHandler(t *testing.T) {
 	if ent.ResponseBodySize != int64(len(responseMsg)) {
 		t.Errorf("ResponseBodySize = %d; want %d", ent.ResponseBodySize, len(responseMsg))
 	}
+	if ent.TraceID != spanCtx.TraceID {
+		t.Errorf("TraceID = %v; want %v", ent.TraceID, spanCtx.TraceID)
+	}
+	if ent.SpanID != spanCtx.SpanID {
+		t.Errorf("SpanID = %v; want %v", ent.SpanID, spanCtx.SpanID)
+	}
+}
+
+type testSpanHandler struct {
+	h       http.Handler
+	spanCtx *trace.SpanContext
+}
+
+func (sh *testSpanHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	ctx, span := trace.StartSpan(r.Context(), "test")
+	defer span.End()
+	r = r.WithContext(ctx)
+	sc := trace.FromContext(ctx).SpanContext()
+	sh.spanCtx = &sc
+	sh.h.ServeHTTP(w, r)
 }
 
-func roundTrip(r *http.Request, h http.Handler) (*Entry, error) {
+func roundTrip(r *http.Request, h http.Handler) (*Entry, trace.SpanContext, error) {
```

Comment:
nit: Seems like this should return a `*trace.SpanContext`?

Reply:
@zombiezen https://github.com/google/go-cloud/pull/1628/commits/e8907ea7eba02b57e4a276f25619f27b534b1a58